moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -319,40 +300,77 @@ void ServoCalcs::stop()
   }
 }
 
+void ServoCalcs::updateParams()
```

Comment:
not necessarily for this PR, but is this where we would choose to add more parameters that we would like to be dynamically settable in future?

If so, worth saying that in the header file docstring for this?

Reply:
> not necessarily for this PR, but is this where we would choose to add more parameters that we would like to be dynamically settable in future?

yes , that is right.
Will add that information in the docstring. Thanks!